components/camera/client_test.go

```diff
@@ -99,13 +96,13 @@ func TestClient(t *testing.T) {
 		ts := time.UnixMilli(12345)
 		return images, resource.ResponseMetadata{CapturedAt: ts}, nil
 	}
-	injectCamera.StreamFunc = func(ctx context.Context, errHandlers ...gostream.ErrorHandler) (gostream.VideoStream, error) {
```

Comment:
given stream still exists & it's behavior hasn't changed, can we please just make these additive changes, rather than removing the Stream tests?

Reply:
Where are the previously existing stream tests?

Reply:
I kept StreamFunc removed because under the hood, the camera client no longer calls this StreamFunc on the underlying camera component